cockatrice/src/tab_server.cpp

```diff
@@ -130,10 +138,16 @@ void RoomSelector::joinClicked()
 
 void RoomSelector::joinFinished(const Response &r, const CommandContainer & /*commandContainer*/, const QVariant &extraData)
 {
-    if (r.response_code() != Response::RespOk)
-        return;
+    if (r.response_code() != Response::RespOk) {
```

Comment:
I think you can add this as a case in the switch to reduce the number of lines and complexity